build/root/.kazelcfg.json

```diff
@@ -3,7 +3,8 @@
 	"SkippedPaths": [
 		"^_.*",
 		"/_",
-		"^third_party/etcd.*"
+		"^third_party/etcd.*",
+		"k8s.io/sample-apiserver/pkg/apis/.*"
```

Comment:
no idea what this is doing... surprised we needed this added here if `staging/src/k8s.io/apiextensions-apiserver/pkg/generated/openapi/zz_generated.openapi.go` didn't require it

Reply:
Oh, I think we can remove it now that Makefile.generated_files is extended.

Reply:
fixed